src/main/java/frc/robot/subsystems/Intake/IntakeConstants.java

```diff
@@ -33,8 +33,8 @@ public enum IntakeState {
 
         final double targetMotorOutput;
 
-        IntakeState(double targetIntakeOutput) {
-            this.targetMotorOutput = targetIntakeOutput;
+        IntakeState(double targetOutput) {
+            this.targetMotorOutput = targetOutput;
```

Comment:
No need for the word target. It won't necacerily be used as a target position everytime